### ISSUE TITLE
No Dense Regression Loss?

### ISSUE THREAD

**Issue by jjjjohnson** [biubug6/Pytorch_Retinaface#86]
请教一下 文章里面花了比较大的篇幅介绍了 dense 3D vertics 但是这个项目里面没有体现，可以说明一下为啥吗？（我猜是有没有这个loss对效果影响不大？） 

多谢！

**Comment by Hongtao-Niro** [comment]
Yeah same question here. I think dense 3D regression loss was one of the main contribution. Can you explain more? Is it because its effect is negligible?
